distributions/openhab-online/readme.md

```diff
@@ -0,0 +1,37 @@
+openHAB online distro
+---------------------
+
+
+The online distro doesn't contain many features.
+You'll need to install those manually for now.
+
+
+1. start openhab
+	start.sh
+
+2. view available features
+	feature:list
+
+3. install a feature e.g. install rest docs
+	feature:install openhab-misc-restdocs
```

Comment:
> My preference would go to extending the dashboard a little bit with a default admin/feature installation page.

This would also allow us to not specify all repository urls by default, but add them programmatically when needed. E.g only install the snapshot repos when the user has selected the 'include snapshots' checkbox in the admin page.